README.md

```diff
@@ -24,39 +23,60 @@ the output is of size as `(nrows, ncols_v)`. If all input tensors are on GPU, a
 
 
 ## Installation
+After setting up an environment with pytorch >= 1.3, run either of these commands from the root folder of the repo: 
 
-Just `python setup.py install`, in the root folder of this repo. This will compile
-and install the torchsearchsorted module.
-be careful that sometimes, `nvcc` needs versions of `gcc` and `g++` that are older than those found by default on the system. If so, just create symbolic links to the right versions in your cuda/bin folder (where `nvcc` is)
+```bash
+pip install -v .
+```
 
-be careful that you need pytorch to be installed on your system. The code was tested on pytorch v1.3
+```bash
+python setup.py install -v
+```
 
-## Usage
+The verbose flag `-v` is not mandatory, but it will print whether the installer was able to find `nvcc` and install the CUDA version of `torchsearchsorted`.
+If you're having problems with the installation, make sure `nvcc` and `gcc` are installed and available in your path, e.g.:
+```bash
+export PATH="/usr/local/cuda/bin:${PATH}"
+export CPATH="/usr/local/cuda/include:${CPATH}"
```

Comment:
This is not enough to have `gcc` and `g++` in the path.
Actually, what is required is to have `nvcc` get to `gcc` and `g++` that are the version it wants.
For me, I had to do the following:

> sudo apt-get install g++-8 gcc-8
> sudo ln -s /usr/bin/gcc-8 /usr/local/cuda-10.1/bin/gcc
> sudo ln -s /usr/bin/g++-8 /usr/local/cuda-10.1/bin/g++

Reply:
Oh that's strange, for me `nvcc` V10.1.105 and `gcc` 7.4.0 work together without issues.

I think the trick with `PATH` and `CPATH` (which is what [torch_scatter](https://github.com/rusty1s/pytorch_scatter) suggests) is safer than creating system-wide links for the simple reason that if some other program relies on specific versions it won't be affected by changing the `PATH` for the current shell.

We can document both approaches in the installation instructions, I think.

Reply:
yes, `gcc` 7 should work. But I had 9 and it doesn't